pyaedt/modules/solutions.py

```diff
@@ -2420,7 +2465,19 @@ def _rotation_to_euler_angles(R):
 
 
 class FfdSolutionDataExporter(FfdSolutionData):
-    """Export far field solution data.
+    """Class to enable export of embedded element pattern data from HFSS.
```

Comment:
```suggestion
    """EnableS export of embedded element pattern data from HFSS.
```
We don't need to indicate that this is a class as its clear here and in the documentation.